designer/server/src/main/scala/pl/touk/nussknacker/ui/server/AkkaHttpBasedRouteProvider.scala

```diff
@@ -200,7 +200,8 @@ class AkkaHttpBasedRouteProvider(
       val newProcessPreparer = processingTypeDataProvider.mapValues { processingTypeData =>
         new NewProcessPreparer(
           processingTypeData.deploymentData.metaDataInitializer,
-          processingTypeData.deploymentData.scenarioPropertiesConfig
+          processingTypeData.deploymentData.scenarioPropertiesConfig,
```

Comment:
Why can't we just somehow merge `deploymentData.scenarioPropertiesConfig` with these from `additionalUIConfigProvider` at the very beginning of the process in the place where they are already merged with properties from config - the same as we do with component's parameters?

Reply:
I guess we could, but this way changes to the return value of `getScenarioPropertiesUIConfigs` take effect immediately, without requiring model reload.

Reply:
wait, let me check some things

Reply:
i've managed to rather easily get rid of this limitation

>   // `ScenarioPropertyConfig.validators` currently does nothing (only usage goes to createUIScenarioPropertyConfig)

including them in ScenarioPropertiesValidator pretty much makes them equal to `normal` scenario properties, so i've also removed the filter from the thread above